src/authorizations/components/CustomApiTokenOverlay.tsx

```diff
@@ -0,0 +1,54 @@
+import React, {FC, useState} from 'react'
```

Comment:
We don't need this component at all. The only one we need to use is the one under `redesigned` folder.